README.md

```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating samples. AMP by Example provides a default styling for common elements (p, h1, h2, h3, a, ...) which you should use. All styles must live in the head of the document using the tag `<style amp-custom>`.
+For AMPByExample, we are using templates, so that we can manage better the CSS, as suggested [here](https://www.ampproject.org/docs/guides/responsive/style_pages.html).
+After you write the example and the page is rendered by gulp, the CSS classes will be added to the AMP page from the existing CSS files.
```

Comment:
Suggestion, rephrase to: If you compile a sample via Gulp and run it, the default styling will be applied.